readme.txt

```diff
@@ -14,11 +14,17 @@ Enable Accelerated Mobile Pages (AMP) on your WordPress site.
 
 This plugin adds support for the [Accelerated Mobile Pages](https://www.ampproject.org) (AMP) Project, which is an open source initiative that aims to provide mobile optimized content that can load instantly everywhere.
 
-With the plugin active, all posts on your site will have dynamically generated AMP-compatible versions, accessible by appending `/amp/` to the end your post URLs. For example, if your post URL is `http://example.com/2016/01/01/amp-on/`, you can access the AMP version at `http://example.com/2016/01/01/amp-on/amp/`. If you do not have [pretty permalinks](https://codex.wordpress.org/Using_Permalinks#mod_rewrite:_.22Pretty_Permalinks.22) enabled, you can do the same thing by appending `?amp=1`, i.e. `http://example.com/?p=123&amp=1`
+With the plugin active, all posts on your site will have AMP-compatible versions, accessible by appending `/amp/` to the end your post URLs. For example, if your post URL is `http://example.com/2016/01/01/amp-on/`, you can access the AMP version at `http://example.com/2016/01/01/amp-on/amp/`. If you do not have [pretty permalinks](https://codex.wordpress.org/Using_Permalinks#mod_rewrite:_.22Pretty_Permalinks.22) enabled, you can do the same thing by appending `?amp=1`, i.e. `http://example.com/?p=123&amp=1`
 
-Note #1: homepage, the blog index, and archives are not currently supported.
+Also, your pages and custom post types can have AMP versions. Simply check their boxes on the 'AMP Settings' page in `/wp-admin`.
 
-Note #2: this plugin only creates AMP content but does not automatically display it to your users when they visit from a mobile device. That is handled by AMP consumers such as Google Search. For more details, see the [AMP Project FAQ](https://www.ampproject.org/docs/support/faqs.html).
+Your entire site can render as "Native AMP" if your theme calls `add_theme_support( 'amp' )`. There will only be one version of each URL: the AMP version. There won't be separate URLs with `/amp` or `?amp` appended. See this [wiki page](https://github.com/Automattic/amp-wp/wiki/Adding-Theme-Support#native-amp) for details and restrictions.
+
+Your theme can also use [Paired Mode](https://github.com/Automattic/amp-wp/wiki/Adding-Theme-Support#paired-mode), with your own custom templates for the AMP URLs.
+
+"Native AMP" and "Paired Mode" add full support for commenting and widgets.
+
+This plugin only creates AMP content but does not automatically display it to your users when they visit from a mobile device. That is handled by AMP consumers such as Google Search. For more details, see the [AMP Project FAQ](https://www.ampproject.org/docs/support/faqs.html).
```

Comment:
This line might cause confusion. It _does_ display the content on mobile if they use the `/amp` or `?amp=1` URL. It appears green in the diff only because I removed `"Note #2:"` at the beginning of it.

Reply:
This is also not true for native/canonical mode, in which AMP content _does_ automatically display for mobile users (and desktop users too). You note this above. So yes I think that this paragraph should note something about if a theme does not support `amp` then basic (and legacy) post templates will be presented to Google Search, Twitter, and other AMP consumers to surface when presenting content to users for discovery. Otherwise, the AMP content is not normally displayed if a user is just navigating around a site.

Reply:
Thanks, @westonruter. How does [this commit](https://github.com/Automattic/amp-wp/pull/1055/commits/178e3161990a8d83ac597153856aa3db967f3241) look?